dev_guide/routes.adoc

```diff
@@ -225,20 +225,42 @@ ifdef::openshift-online[]
 [[custom-route-and-hosts-and-certificates-restrictions]]
 == Restrictions
 
-Custom route hosts are not permitted. Instead, the following host template is
-enforced on all user routes:
+Routes are restricted in {product-title} Starter, but are not restricted in
+{product-title} Pro. Custom route hosts are permitted in {product-title} Pro. If
+using {product-title} Starter, the following host template is enforced on all
+user routes:
+
+----
+<route-name>-<namespace>.<external-address>
+----
+
+For example:
 
 ----
 <route-name>-<namespace>.1d35.starter-us-east-1.openshiftapps.com
 ----
 
-Also, custom certificates are not permitted. Only unencrypted routes, edge
-routes using the default certificate, and passthrough routes work. Edge routes
-with custom certificates and re-encrypt routes (which necessarily have custom
-certificates) do not work.
+To determine the external address, run:
+
+----
+$ oc get route/<route-name>
+----
+
+Custom certificates are permitted in {product-title} Pro. In {product-title}
+Starter, only unencrypted routes, edge routes using the default certificate, and
+passthrough routes work. Edge routes with custom certificates and re-encrypt
+routes (which necessarily have custom certificates) do not work in
+{product-title} Starter.
 
 These restrictions are enforced in the router. Inspecting the route in the
 console or using `oc get routes` displays the host and certificates specified by
-the user. However, custom hosts are not respected, and routes with custom
-certificates do not work.
+the user. However, custom hosts are not respected in {product-title} Starter,
+and routes with custom certificates do not work.
+
+Once your custom route is created, you must update your DNS provider by creating
+a canonical name (CNAME) record. Your CNAME record should point your custom
+domain to the {product-title} router as the alias. Your {product-title} router
+is the same per-cluster issue that i son {product-title} Starter. It is
+different for every region. CNAME records can not be set for a naked domain
```

Comment:
s/i son/is on/

s/can not/cannot/

I'd put the last two sentences about naked domains as a separate paragraph, maybe notated with `[NOTE]`.

I don't understand the sentences, "Your {product-title} router is the same per-cluster issue that i son {product-title} Starter. It is different for every region." I am guessing this could be replaced simply by, "The {product-title} router's domain is different for every cluster."